src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveAttributeCompletionItemProvider.cs

```diff
@@ -88,8 +97,10 @@ internal static ImmutableArray<RazorCompletionItem> GetAttributeCompletions(
         }
 
         // Use ordinal dictionary because attributes are case sensitive when matching
-        using var _ = SpecializedPools.GetPooledStringDictionary<(ImmutableArray<BoundAttributeDescriptionInfo>, ImmutableArray<RazorCommitCharacter>)>(out var attributeCompletions);
-        var inSnippetContext = InSnippetContext(containingAttribute, razorCompletionOptions);
+        using var _ = SpecializedPools.GetPooledStringDictionary<(ImmutableArray<BoundAttributeDescriptionInfo>, ImmutableArray<RazorCommitCharacter>, RazorCompletionItemKind kind)>(out var attributeCompletions);
+
+        // Collect indexer descriptors and their parent tag helper type names
```

Comment:
I have no idea what an indexer descriptor is, and I'm guessing you didn't before you started working in this file. Assuming you've had to find out to do this PR, could you put a comment somewhere explaining (unless its already somewhere not shown in the diff)